app/controllers/v0/institution_programs_controller.rb

```diff
@@ -81,7 +81,14 @@ def filter_results(relation)
         relation = relation.filter(filter_args[0], @query[filter_args[1]])
       end
 
-      relation = relation.where(institutions: { count_of_caution_flags: 0 }) if @query[:exclude_caution_flags]
+      if @query[:exclude_warnings]
+        # prod flag for bah-8187. Remove are approval.
```

Comment:
Could you clarify this comment for me?

Reply:
Might have been a typo. The idea was to keep us from using that filter in prod. But I just realized that we still need the old filter so I've added it back in and removed the prod flag .